src/main/java/Output.java

```diff
@@ -0,0 +1,22 @@
+public class Output {
```

Comment:
I like how you extracted the various System out print methods into a separate class